koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -82,27 +83,27 @@ def _get_tenant_provider(self):
 
     def schema_exists(self):
         """Check if schema exists."""
+        LOG.info(log_json(msg="Checking for schema", schema=self._schema_name))
```

Comment:
```suggestion
        LOG.info(log_json(msg="checking for schema", schema=self._schema_name))
```
I've been trying to eliminate as much capitalization and punctuation as possible. It makes it easier to assert logs if we need to since there is less thinking about proper grammer.

Reply:
This is good to know!